sdk/cosmos/azure_data_cosmos/README.md

```diff
@@ -1,5 +1,155 @@
 # Azure Cosmos DB SDK for Rust.
 
-## Introduction
+This client library enables client applications to connect to Azure Cosmos DB via the NoSQL API. Azure Cosmos DB is a globally distributed, multi-model database service.
 
-This client library enables client applications to connect to [Azure Cosmos DB](https://azure.microsoft.com/en-us/products/cosmos-db) via the NoSQL API. Azure Cosmos DB is a globally distributed, multi-model database service.
+[Source code] | [Package (crates.io)] | [API reference documentation] | [Azure Cosmos DB for NoSQL documentation]
+
+## Getting started
+
+### Install the package
+
+Install the Azure Cosmos DB SDK for Rust with cargo:
+
+```sh
+cargo add azure_data_cosmos
+```
+
+### Prerequisites
+
+* An [Azure subscription] or free Azure Cosmos DB trial account.
+
+Note: If you don't have an Azure subscription, create a free account before you begin.
+You can Try Azure Cosmos DB for free without an Azure subscription, free of charge and commitments, or create an Azure Cosmos DB free tier account, with the first 400 RU/s and 5 GB of storage for free. You can also use the Azure Cosmos DB Emulator with a URI of https://localhost:8081. For the key to use with the emulator, see [how to develop with the emulator](https://learn.microsoft.com/azure/cosmos-db/how-to-develop-emulator).
+
+### Create an Azure Cosmos DB account
+
+You can create an Azure Cosmos DB account using:
+
+* [Azure Portal](https://portal.azure.com).
+* [Azure CLI](https://learn.microsoft.com/cli/azure).
+* [Azure ARM](https://learn.microsoft.com/azure/cosmos-db/quick-create-template).
+
+#### Authenticate the client
+
+In order to interact with the Azure Cosmos DB service you'll need to create an instance of the `CosmosClient` struct. To make this possible you will need a URL and key of the Azure Cosmos DB service.
+
+## Examples
+
+The following section provides several code snippets covering some of the most common Azure Cosmos DB NoSQL API tasks, including:
+* [Create Client](#create-cosmos-db-client "Create Cosmos DB client")
+* [Create Database](#create-database "Create Database")
+* [Create Container](#create-container "Create Container")
```

Comment:
Looks like these links no longer resolve.

Reply:
Yep, I forgot to remove the TOC items. Thanks!